layers/+completion/ivy/config.el

```diff
@@ -13,7 +13,10 @@
 ;; Variables
 
 (defvar spacemacs--counsel-commands
-  '(("rg" . "rg --smart-case --no-heading --color never %s %S .")
+  '(;; -n forces line numbers (disabled by default on windows)
+    ;; no --vimgrep because it adds column numbers that wgrep can't handle
+    ;; (see https://github.com/syl20bnr/spacemacs/pull/8065)
+    ("rg" . "rg --smart-case --no-heading --color never -n %s %S .")
     ("ag" . "ag --nocolor --nogroup %s %S .")
     ("pt" . "pt -e --nocolor --nogroup %s %S .")
     ("ack" . "ack --nocolor --nogroup %s %S .")
```

Comment:
⬇️  There's a typo down here "Alist" if you want to fix that up while you're in here